docs/tutorials/gp-classification.ipynb

```diff
@@ -0,0 +1,386 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
```

Comment:
This should include a `hide-cell` tag. Take a look at some of the other tutorials.